setup.py

```diff
@@ -38,6 +38,7 @@
 import pkg_resources
 from Cython.Build import cythonize
 
+import platform
```

Comment:
Could you put this import with others (https://github.com/fofix/fofix/blob/master/setup.py#L29) ? Thank you :).